todo/src/index.js

```diff
@@ -0,0 +1,101 @@
+import StorageController from "./scripts/storageController";
+import themeToggler from "./scripts/themeToggler";
+import * as sortBarView from  "./scripts/sortBarView";
+import * as tasksFieldView from "./scripts/tasksFieldView";
+import * as modalForm from "./scripts/modalForm";
+import generateId from "./scripts/idGenerator";
+import setCounter from "./scripts/itemCounter";
+import * as dictionary from "./scripts/dictionary";
+import "./style/index.scss";
+import "./style/toggler.scss";
+
+themeToggler();
+
+class ToDo {
+    constructor(){
+        this.storageController = new StorageController();
+        modalForm.addSubmitHandler(this.modalSubmitCallback.bind(this));
+        tasksFieldView.setTasksContainerClickHandler(this.changeTaskMode.bind(this));
+        this.sortDirection = this.storageController.getSortDirection();
+        sortBarView.initChangeDirectionHandler(this.sortDirection, this.changeTaskSortDirection.bind(this));
+        this.sortDirection = this.storageController.getSortDirection();
+        tasksFieldView.setSortDirection(this.sortDirection);
+        this.refreshTasksField();
+    }
+
+    changeTaskSortDirection(isChange) {
+        if (isChange) {
+            tasksFieldView.setSortDirection(this.storageController.changeSortDirection());
+            this.refreshTasksField();
+        }
+    }
+
+    refreshTasksField() {
+        const tasksData = this.storageController.getItemsList();
```

Comment:
use spread operator `const [old, new] =...`

Reply:
I used spread operator.
```suggestion
    refreshTasksField() {
        const [currentTasks, completeTasks] = this.storageController.getItemsList();
        tasksFieldView.refreshTaskFields(currentTasks, completeTasks);
        setCounter(currentTasks.length, completeTasks.length);
    }
```